src/test/java/com/google/api/codegen/testdata/python_doc_main_library.baseline

```diff
@@ -1473,9 +1473,9 @@ class LibraryServiceClient(object):
           >>> required_repeated_string = []
           >>> required_repeated_bytes = []
           >>> required_repeated_message = []
-          >>> required_repeated_resource_name = api.book_path('[SHELF_ID]', '[BOOK_ID]')
-          >>> required_repeated_resource_name_oneof = api.book_path('[SHELF_ID]', '[BOOK_ID]')
-          >>> required_map = []
+          >>> required_repeated_resource_name = []
```

Comment:
It's repeated; it should be a list. Not sure why it wasn't a list in the first place.